orchagent/portsorch.cpp

```diff
@@ -8412,7 +8365,19 @@ void PortsOrch::addWredQueueFlexCountersPerPortPerQueueIndex(const Port& port, s
         queue_ids = port.m_queue_ids;
     }
 
-    wred_queue_stat_manager.setCounterIdList(queue_ids[queueIndex], CounterType::QUEUE, counter_stats);
+    wred_queue_stat_manager.setCounterIdList(queue_ids[queueIndex], CounterType::QUEUE, counter_stats, queueType);
+}
+
+void PortsOrch::flushCounters()
+{
+    port_stat_manager.flush();
+    port_buffer_drop_stat_manager.flush();
+    queue_stat_manager.flush();
+    queue_watermark_manager.flush();
+    pg_watermark_manager.flush();
+    pg_drop_stat_manager.flush();
+    wred_port_stat_manager.flush();
+    wred_queue_stat_manager.flush();
```

Comment:
should those be in some set or vector if they share the same base ? so we could use or loop to flush them ?